samples/spm/README.rst

```diff
@@ -11,7 +11,8 @@ The Secure Partition Manager sample provides a reference use of the System Prote
 This firmware sets up an nRF device with Trusted Execution (|trusted_execution|) so that it can run user applications in the non-secure domain.
 
 .. note::
-   SPM is an alternative for using the Trusted Firmware-M (TF-M). See :ref:`ug_tfm`.
+   SPM is now DEPRECATED and will be removed in a future version of the SDK.
```

Comment:
```suggestion
   SPM is deprecated as of |NCS| v2.1.0 and will be removed in a future version of the SDK.
```
We should probably mention the version of NCS it is deprecated in, right?

Reply:
Done